.github/CODEOWNERS

```diff
@@ -88,6 +88,7 @@
 /src/legacy/server/status/ @elastic/kibana-platform
 
 # Security
+/src/core/server/csp/  @elastic/kibana-security
```

Comment:
Will GH ping us on changes? Both teams were owners before https://github.com/elastic/kibana/pull/52698/files#diff-5cfb934ca0ec4c1743905a51dc428b84R83

Reply:
Yep! Platform is already an owner due to L77:

https://github.com/elastic/kibana/pull/52698/files/7ab6d996c76884a170ecd3961e458fdd5686a400#diff-5cfb934ca0ec4c1743905a51dc428b84R77

Reply:
ok. there are 2 problems
- /src/legacy/server/csp/ @elastic/kibana-platform wasn't removed
- let's check later whether it works because I found ` The last matching pattern has the most precedence.` https://github.blog/2017-07-06-introducing-code-owners/